app/controllers/application_controller.rb

```diff
@@ -4,10 +4,19 @@ class ApplicationController < ActionController::Base
   before_action :authenticate_user!
 
   def after_sign_in_path_for(resource)
+    # Path to change passwords after first login
```

Comment:
Bisogna gestire il caso in cui si vada nella pagina di reset senza aggiornare la password